### ISSUE TITLE
Params are not properly generated for java classes.

### ISSUE THREAD

**Issue by mp2019** [Kotlin/dokka#465]
Given an example class:

```java
package com.test.app;

/**
 * Provide access to the service.
 */
public class Login {

        /** Perform the loging
         *
         * @param name user name
         * @param password user password
         */
        public void login(String name, String password) {
        }

}
```

for configuration:

```
task dokkaJavadoc(type: org.jetbrains.dokka.gradle.DokkaTask) {
	outputFormat = 'javadoc'
}
```

The output param descriptions contains not expected characters:

```
public void login(java.lang.String name,
                  java.lang.String password)

Perform the loging

Parameters:
    name - org.jetbrains.dokka.NodeRenderContent@1f2d9c1d: user name
    password - org.jetbrains.dokka.NodeRenderContent@775ef0fa: user password 
```

Dokka version: 0.9.18

**Comment by cclover** [comment]
I have met the same issue for java class.  The kotlin class show normally.

**Comment by GCoder12** [comment]
I'm having the same issue on latest version of Dokka, 0.9.18

**Comment by KrystianUjma** [comment]
I fixed it and pushed on [multiplatform branch](https://github.com/Kotlin/dokka/tree/multiplatform-support). Will be released in 0.9.19.
